sessions/react-basics/demo-render/src/index.js

```diff
@@ -9,4 +9,11 @@ const root = createRoot(rootElement);
 
 // Call render on the root to render the elements into the DOM.
 // Notice that this is NOT a template literal, it is JSX.
-root.render(<h1>Hello World 👋</h1>);
+root.render(
+  <h1>
+    Hello World{" "}
+    <span role="img" aria-label="Waving hand">
+      👋
+    </span>
+  </h1>
+);
```

Comment:
This is the very first line of React code the students see. To keep it super simple, I'd just skio the emoji here.

➡️ We need to adjust the sesion guide aswell

```suggestion
root.render(<h1>Hello World</h1>);
```